packages/firestore/src/local/shared_client_state.ts

```diff
@@ -711,6 +719,10 @@ export class WebStorageSharedClientState implements SharedClientState {
     this.persistOnlineState(onlineState);
   }
 
+  remoteDocumentsChanged(): void {
```

Comment:
`notifyRemoteDocumentsChanged()`? But also see my comment below: I think this should be bundle specific.